### PR TITLE
Test reindexing with two content clusters sharing cluster-controller

### PR DESCRIPTION
**Pull request by bjorncs**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: tests/search/reindexing/reindexing.rb [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+  def setup
+    set_owner('bjorncs')
+  end
+
+  def test_reindexing_with_multiple_content_clusters
+    app = SearchApp.new.monitoring('vespa', 60).
+        cluster(SearchCluster.new(MUSIC_CLUSTER_ID).sd(selfdir + 'music.sd')).
+        cluster(SearchCluster.new(MOVIE_CLUSTER_ID).sd(selfdir + 'movie.sd')).
+        container(Container.new('combinedcontainer').
+            jvmargs('-Xms16g -Xmx16g').
```
Comment (jonmv) [inline tests/search/reindexing/reindexing.rb:21]: Possibly a bit high for a non-performance-test. 
```suggestion
```

Comment (bjorncs) [reply tests/search/reindexing/reindexing.rb]: Fixed :)

**Comment by bjorncs** [comment]
@jonmv PTAL

**Review by jonmv — APPROVED** [review]

**Comment by jonmv** [comment]
🦷

**Pull request merged**